packages/backend/src/main.ts

```diff
@@ -2,8 +2,15 @@ import { ConfigService } from '@nestjs/config';
 import { NestFactory } from '@nestjs/core';
 import { DocumentBuilder, SwaggerModule } from '@nestjs/swagger';
 import { AppModule } from './app.module';
+import configureEnvironmentFiles from './scripts/configure-env';
 
 async function bootstrap() {
+  // Verifying the configuration of environment files, direct return if the result is not true
+  const isValidated = configureEnvironmentFiles(process.env.NODE_ENV);
```

Comment:
Should we just export `validateEnvironmentFile` so that we are not validating both environments when starting? I don't care about prod env when i'm starting dev env & vice-versa. Should be fine though.

Reply:
The called function only validates based on the env passed in.

Reply:
ohh my bad